### DISCUSSION TITLE
How can I run Itinerary from source?

### DISCUSSION THREAD

**Discussion by redstone-dev** [Looky1173/Itinerary#22] [Q&A] [answered]
I'm translating the strings to French but I want to see if they work. So I want to clone my fork and build it from source to see what it looks like. How do I do that?

**Comment by Looky1173** [comment, marked as answer]
Once you've cloned the repository, install the dependencies and run `yarn dev` or `npm dev` depending on your package manager. The parts of the site that communicate with the backend won't work because they require [Itinerary-Server](https://github.com/Looky1173/Itinerary-Server).

Anyway, if you have your translation file ready (`fr-FR.js`), feel free to open a pull request and I will approve the changes. **There's nothing else you need to do to get the French translation working besides creating `fr-FR.js`.**

Thanks for translating Itinerary! 😀

**Reply by redstone-dev** [reply]
Thanks! Also, I'm not yet done the translation, there's still lots more to be done. But when I'm done I'll create a PR :smile:
